<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPiciFrom</class>
 <widget class="QWidget" name="QPiciFrom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_pici_pgdn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_pici_save">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_pici_delete">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_pici_back">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_pici_pgup">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed_pici_name">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed_pici_shuliang">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed_pici_tiji">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed_pici_suannongdu">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_pici_leixing">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_pici_danwei">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pici_yangpinmingcheng">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pici_shuliang">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pici_danwei">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pici_tiji">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pici_leixing">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pici_nongdu">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pici_ml">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pici_mol">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableView" name="tb_pici_param">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::CurrentChanged|QAbstractItemView::EditKeyPressed</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_pici_yangpinshuliang">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>340</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>pb_pici_pgdn</zorder>
  <zorder>pb_pici_save</zorder>
  <zorder>pb_pici_delete</zorder>
  <zorder>pb_pici_back</zorder>
  <zorder>pb_pici_pgup</zorder>
  <zorder>ed_pici_name</zorder>
  <zorder>ed_pici_shuliang</zorder>
  <zorder>ed_pici_tiji</zorder>
  <zorder>ed_pici_suannongdu</zorder>
  <zorder>cb_pici_leixing</zorder>
  <zorder>cb_pici_danwei</zorder>
  <zorder>lb_pici_yangpinmingcheng</zorder>
  <zorder>lb_pici_shuliang</zorder>
  <zorder>lb_pici_danwei</zorder>
  <zorder>lb_pici_tiji</zorder>
  <zorder>lb_pici_leixing</zorder>
  <zorder>lb_pici_nongdu</zorder>
  <zorder>lb_pici_ml</zorder>
  <zorder>lb_pici_mol</zorder>
  <zorder>cb_pici_yangpinshuliang</zorder>
  <zorder>label</zorder>
  <zorder>tb_pici_param</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
